.changeset/happy-chairs-ring.md

```diff
@@ -0,0 +1,7 @@
+---
+"@salt-ds/lab": minor
+---
+
+When using the `stepped-tracker` with the `useStepReducer` hook, there was an edge case where the state indicated that all steps were completed, but the `ended` boolean remained `false`. This issue stemmed from an immutability problem in the state handling logic.This bug is resolved now, whilst ensuring proper immutability and fixing the `ended` boolean behavior. Additionally, the `reset` action has been introduced, which, as the name suggests, resets the state so that the first step becomes active and all others are marked as pending.
```

Comment:
Changset will become changelog / release note for our consumers, thus our standard starting with "Fixes", "Added", etc. 

Half of this paragraph fits more nicely within PR description and/or site documentation...?

Reply:
Done!